modules/navier_stokes/src/physics/WCNSFVFluidHeatTransferPhysics.C

```diff
@@ -403,16 +482,67 @@ WCNSFVFluidHeatTransferPhysics::addEnergyWallBC()
   }
 }
 
+void
+WCNSFVFluidHeatTransferPhysics::addMaterials()
+{
+  if (!_has_energy_equation)
+    return;
+
+  // Note that this material choice would not work for Newton-INSFV + solve_for_enthalpy
```

Comment:
An explanation of why not would be appreciated

Reply:
Especially because the material name is `INSFVEnthalpyFunctorMaterial` I would expect full support for (Newton-)INSFV

Reply:
It's to compute the enthalpy from temperature, which one should not need to do when solving for enthalpy